api/enki.py

```diff
@@ -773,23 +740,21 @@ def process_book(self, bibliographic, availability):
             license_pool.collection,
             replace=policy,
         )
-        if new_edition:
-            bibliographic.apply(edition, self.collection, replace=policy)
+        bibliographic.apply(edition, self.collection, replace=policy)
+        if not license_pool.work:
+            work, is_new = license_pool.calculate_work(
+                even_if_no_author=True
+            )
+            if work:
+                work.set_presentation_ready()
```

Comment:
Here's the code I had to add because I got rid of the BibliographicCoverageProvider.